install_scripts/geo-concerns.sh

```diff
@@ -24,5 +20,5 @@ yes | cp -rf $SHARED_DIR/install_scripts/Rakefile .
 # Make solr core nonpersistent. Causes errors when solr-wrapper is restarted.
 sed -i 's/persist: true/persist: false/g' .solr_wrapper
 
-# setup fedora and solr
-rake demo:setup
+# Run the servers
+rake demo:servers
```

Comment:
Pretty sure we want to convert this back to `rake demo:setup`. The user needs to ssh into the box run a rake task to get everything going.